packages/lib-classifier/src/store/WorkflowStepStore.spec.js

```diff
@@ -173,6 +173,82 @@ describe('Model > WorkflowStepStore', function () {
     })
   })
 
+  describe('when the workflow tasks define next steps as tasks (backwards compatibility)', function () {
```

Comment:
This largely copied from the previous describe block, not sure the following tests are really necessary, erred on side of including, though happy to remove if prefer:
- it('should assign each task to a step'
- it('should set the first step to be active'
- it('should generate a step from the combo task'